sea-orm-rocket/lib/Cargo.toml

```diff
@@ -13,19 +13,19 @@ edition = "2021"
 all-features = true
 
 [dependencies.rocket]
-version = "0.5.0"
+version = "0.5"
 default-features = false
 
 [dependencies.sea-orm-rocket-codegen]
 path = "../codegen"
-version = "0.5.1"
+version = "0.5"
```

Comment:
0.5.5

Reply:
I'm not sure what you mean. `sea-orm-rocket-codegen` is on `0.5.4`.

```
[package]
name = "sea-orm-rocket-codegen"
version = "0.5.4"
```

Should I increment the version? Or is this a note for yourself?
(Can I still change something as this was already merged?)

Reply:
it's a note to myself to bump the version haha